lighthouse-core/audits/critical-request-chains.js

```diff
@@ -132,8 +134,14 @@ class CriticalRequestChains extends Audit {
             formatter: Formatter.SUPPORTED_FORMATS.CRITICAL_REQUEST_CHAINS,
             value: {
               chains,
-              longestChain: CriticalRequestChains._getLongestChain(chains)
+              longestChain
             }
+          },
+          details: {
+            type: 'crc',
```

Comment:
can we avoid abbreviations here, there are other ways to reduce LHR size I'd want to start with :)